abis_mapping/templates/survey_occurrence_data_v2/mapping.py

```diff
@@ -647,15 +638,25 @@ def apply_mapping_row(
             sensitivity_category_value = None
             sensitivity_category_collection = None
 
+        # Create URIs for Survey-related fields (i.e. fields not on the incidental template)
+
         # Create TERN survey IRI from surveyID field
         survey_id: str | None = row["surveyID"]
         survey = utils.iri_patterns.survey_iri(base_iri, survey_id)
 
+        # Create Tern Site IRI, depending on the siteID field
+        site_id: str | None = row["siteID"]
+        if site_id:
```

Comment:
not a walrus enjoyer?

Reply:
sometimes yes, but here the walrus was preventing me from adding a proper type annotation to the variables like `site_id`

Reply:
i probs didn't notice its use somewhere mypy complained.

Reply:
Well without the type annotation `row["field"]` is `Any`, and `Any` means "don't type check this", so mypy generally won't complain about that